go/cmd/mysql_probe/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/coreos/go-systemd/daemon"
+	"github.com/gorilla/mux"
+	"github.com/inverse-inc/go-utils/log"
+	"github.com/inverse-inc/packetfence/go/galeraautofix/mariadb"
+)
+
+var ctx = context.Background()
+
+func main() {
+
+	router := mux.NewRouter()
+	router.HandleFunc("/", checkDB).Methods("OPTIONS")
+
+	srv := &http.Server{
+		Addr:        ":3307",
+		IdleTimeout: 5 * time.Second,
+		Handler:     router,
+	}
+	// Systemd
+	daemon.SdNotify(false, "READY=1")
+
+	go func() {
```

Comment:
Looks like if there is a panic in this block, this service will die. Is that what we want?

Not saying its bad or good, just want to make sure we're happy with how it will behave

Reply:
I want to test if the service still reply, if it dies then systemd will restart it.